esp-hal/src/lib.rs

```diff
@@ -325,7 +325,6 @@ unstable_driver! {
     pub mod aes;
     #[cfg(assist_debug)]
     pub mod assist_debug;
-    #[cfg(any(xtensa, all(riscv, systimer)))]
```

Comment:
Removed this bit of nonsense, `delay` doesn't use any hardware timers directly, just `esp_hal::time`